external-events/txr/txr.public.output.transactions.v1

```diff
@@ -0,0 +1,8 @@
+{
+  "$id": "txr.public.output.transactions.v1",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "SafPublishes a raw transaction event.",
+  "description": "Publishes a raw transaction event.",
```

Comment:
update description with more info like: binary payload is zip and poslog xml whatever.